docusaurus/docs/migration-guides/angular-react/angular-react-custom-configuration-components.md

```diff
@@ -0,0 +1,62 @@
+---
+id: custom-config-components
+title: Custom configuration components
+sidebar_position: 4
+description: How to migrate a Grafana plugin from AngularJS with custom fields to React with custom components.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - React
+  - ReactJS
+  - Angular
+  - migration
+---
+
+# AngularJS to React: Custom configuration components
+
+## Background
+
+The Grafana SDK provides many useful configuration components for developers like entering number ranges, thresholds, unit selections, and color selections. You can see descriptions of these elements in the [Grafana design system](https://grafana.com/developers/saga/about/overview).
```

Comment:
Add link to "Grafana SDK". Do you mean the SDK for Go?

Reply:
I was confused too. I think this should be called `@grafana/ui` or `Grafana design system` as https://developers.grafana.com/ui/latest/index.html?path=/story/docs-overview-intro--page calls it

Reply:
yes `@grafana/ui` was the intention here, not the go sdk, will make it clear thanks!